crossbeam/datasets/run_deepcoder_gen.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+data_dir=$HOME/xlambda-data/deepcoder
+
+tout=1800
+maxw=100  # Run until timeout.
+maxne=5
+maxni=3
+skip=0.75
+lambdaskip=0.5
+num_proc=1  # TODO(hadai): change to whatever is reasonable
```

Comment:
@Hanjun-Dai did you change this when you ran dataset generation? If so, can you commit that code? Thanks!

Reply:
Yes but 1) it is machine dependent; 2) I found it is pretty slow and expensive and VMs crashed, I'm figuring out other ways to do it (e.g, use xcloud) so I'll leave this code untouched for now.